DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/JobProfileSection/Index.cshtml

```diff
@@ -5,4 +5,9 @@
     @Html.Raw(Model.TopSectionContent)
     @Html.Raw(Model.PropertyValue)
     @Html.Raw(Model.BottomSectionContent)
-</section>
\ No newline at end of file
+</section>
+
+@if (Model.IsWhatItTakesCadView)
```

Comment:
Is this correct? So for WIT, then there are no wrapping section? 
I thought the markup would be
<section id=wit>
<section id=skills>skills stuff</section>
<section id=restrictions>restrictions and other requirements</section>
</section>